.github/workflows/tools.yml

```diff
@@ -109,3 +109,15 @@ jobs:
     - name: Run Pylint on RDF Generator
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/rdf_generator
+      #---------- ABEL ----------#
+    - name: Run ABEL install dependencies
+      run: |
+        python setup.py install
+      working-directory: ./tools/abel
+    - name: ABEL tests
+      run: |
+        python -m unittest discover -p '*_test.py' --failfast -v
+      working-directory: ./tools/abel/tests
```

Comment:
break this out to create ABEL's workflow?

Reply:
or submit this workflow first to make sure the tests are running ok?

Reply:
Will do. Opened https://github.com/google/digitalbuildings/pull/776

Reply:
Done.